src/test/java/com/microsoft/sqlserver/jdbc/connection/TimeoutTest.java

```diff
@@ -118,7 +118,7 @@ public void testDMLoginTimeoutNotApplied() {
     public void testConnectRetryBadServer() {
         long timerEnd = 0;
         long timerStart = System.currentTimeMillis();
-        int loginTimeout = 1;
+        int loginTimeout = 15;
```

Comment:
Why this timeout is increased here?

Reply:
this caused an intermittent failure as depending on the network 1s is likely not enough time for socket to time out so it test will fail intermittently if it's checking to see if timeout is < 1s